bin/Powheg/production/Run3/13p6TeV/Higgs/ggHtoZZto2L2Q/gg_H_quark-mass-effects_ZZto2L2Q_NNPDF31_13p6TeV_M125.input

```diff
@@ -40,11 +38,11 @@ iseed SEED
 ! **** Mandatory parameters for ALL models ****
 massren 0           ! Mass renormalization scheme. 0 = OS, 1 = MSBAR , 2 = DRBAR
 zerowidth 0         ! Control if the Higgs boson is to be produced on-shell or not: 1 = On-Shell; 0 = Off-shell with Breit-Wigner
-ew 0                ! ew = 0 disable EW corrections - ew = 1 enable EW corrections
+ew 1                ! ew = 0 disable EW corrections - ew = 1 enable EW corrections
 model 0
 gfermi 0.116637D-04        ! GF
 hdecaymode -1      ! PDG code for first decay product of the higgs
-masswindow 9999d0  !(default 10d0) number of widths around hmass in the BW for an off-shell Higgs boson
+masswindow 1d0  !(default 10d0) number of widths around hmass in the BW for an off-shell Higgs boson
```

Comment:
Why did you change the masswindow? 
In principle you can refer to the already existing cards for Powheg ggH, e.g.:
https://github.com/cms-sw/genproductions/blob/master/bin/Powheg/production/Run3/13p6TeV/Higgs/gg_H_quark-mass-effects_with_JHUGen_NNPDF31_13p6TeV/gg_H_quark-mass-effects_NNPDF31_13p6TeV_M125.input

Actually, if you just want a different decay mode, I think you can reuse exactly the same Powheg card and just change the JHUGen card according to the decay channel you want.